Application Examples/_Tutorial/Tutorial/Copyright.mo

```diff
@@ -1,32 +1,14 @@
-within OpenIPSL.Connectors;
-
-
-connector PwPin "connector for electrical blocks treating voltage and current as complex variables"
-  Real vr;
-  // real part of the voltage
-  Real vi;
-  // imaginary part of the voltage
-  flow Real ir;
-  // real part of the current
-  flow Real ii;
-  // imaginary part of the current
-  annotation (Icon(graphics={Rectangle(
-          extent={{-100,98},{100,-102}},
-          lineColor={0,0,255},
-          fillColor={0,0,255},
-          fillPattern=FillPattern.Solid)}), Documentation(revisions="<html>
-<!--DISCLAIMER-->
+within Tutorial;
```

Comment:
Funny GIT registered this as a file change instead of a new file., but I guess it won't matter as the whole file is rewritten